### PR TITLE
Updating sambamba merge tool: Quick fix for ERGA BGE Galaxy workflow

### PR DESCRIPTION
**Pull request by SaimMomin12**
Sorry for another PR, earlier one had some issue with Git branching 

--
Older Text:

Hi @bgruening,

Considering the present state of samtools merge, I think we require more time and efforts to fix and cleanup the wrapper.

In the meantime, until the above is fixed I would propose that we use sambamba merge to circumnavigate the present issues faced in the ERGA BGE Galaxy workflow. With my recent talk with Diego/Tom, they would like to present the ERGA BGE Galaxy workflow in the upcoming GCC. Keep this in mind, I thought of spinning off the sambamba merge to suffice the urgency.

I realized there was already a sambamba tool suite wrapper in your repository developed several years back, but was not put into the EU Galaxy server. I've made some minor changes just in sambamba_merge, updated the tool version and included the relevant test case.

I hope this will help us to tackle the issue just for time being.

TL;DR I've updated the sambamba merge wrapper, included relevant test cases and test the tool. This is quick fix until the samtools merge is fixed. 😃

Thanks!

### CODE REVIEW

**Comment by SaimMomin12** [comment]
It looks like other tools from this suite needs to be updated/fixed too. What do you suggest? I just need the sambamba merge though! 😅

**Comment by SaimMomin12** [comment]
Closing this PR and continued here https://github.com/bgruening/galaxytools/pull/1438
